README.md

```diff
@@ -1,23 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Sewing Pattern Maker
 
-## Available Scripts
+Our web app allows users to generate a sewing pattern from a  photo of an article of clothing.
 
-In the project directory, you can run:
+A user uploads a picture of a garment laying flat on a surface. Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools, as well as providing a measurement. Finally, our app automatically generates a printable pdf pattern that will allow the user to sew a copy of the garment in the fabric of their choice!
```

Comment:
I think "Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools, as well as providing a measurement." should be "Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools. Next, they provide a measurement for a single seam of the original garment."

pdf should PDF

Reply:
I was trying to keep it short, but I agree it maybe wouldn't be clear to a first-time reader. I added your suggestion.